ms2deepscore/data_generators.py

```diff
@@ -4,15 +4,19 @@
 import pandas as pd
 from tensorflow.keras.utils import Sequence
 
+# Set random seed for reproducibility
+np.random.seed(42)
```

Comment:
This will ensure anyone trying to reproduce your results will get exactly the same!

Reply:
Good point. thanks!